src/main/resources/org/biouno/unochoice/stapler/unochoice/UnoChoice.es6

```diff
@@ -658,6 +665,10 @@ var UnoChoice = UnoChoice || (jQuery3 => {
                             let idValue = `ecp_${e.target.randomName}_${i}`;
                             idValue = idValue.replace(' ', '_');
 
+                            let tdClass =
```

Comment:
Not sure if this is correct.
As the if from line 662 applies this to checkboxes I have the impression that a lot of the code here might be obsolete.
Can the variable `entry` ever be something else than an input of type `checkbox` here?  If yes how do I need to setup the parameters to achieve this, e.g. how to make `entry` a string or a JSON?

Reply:
I think in the old Jenkins UI it could have used an input in checkboxes or radio boxes (that's what my memory tells me). But that's probably not the case anymore, for many months/years now.